azurerm/resource_arm_kusto_cluster.go

```diff
@@ -259,7 +259,7 @@ func expandKustoClusterSku(d *schema.ResourceData) (*kusto.AzureSku, error) {
 
 	azureSku := &kusto.AzureSku{
 		Name:     kusto.AzureSkuName(name),
-		Tier:     &tier,
+		Tier:     kusto.AzureSkuTier(tier),
```

Comment:
since there's now a type for this could we switch the validation function (above) to using those constants?